examples/panel/rag-news/rag_news.py

```diff
@@ -3,8 +3,14 @@
 
 from gnews import GNews
 from openai import OpenAI
+import os
+from dotenv import load_dotenv
 
-client = OpenAI()
+load_dotenv(".env")
+openaikey = os.getenv("OPENAI_API_KEY")
```

Comment:
Do we need to use `load_dotenv`? Please refer: https://docs.cloud.ploomber.io/en/latest/user-guide/cli.html#defining-environment-variables

Reply:
yeah, and this is also a different example. there's no need to modify existing examples

Reply:
removed from readme 

https://github.com/ploomber/doc/pull/113/commits/1331ff8941bc7ecaa15d34618d7f40b0a4bc8ad1

and from code

Reply:
tried restoring the code as it was by bringing the copy from Main but the change still appears
how can I remove a specific file from being tracked in a PR

Reply:
Try adding a newline at the end of the file